liquidator/OneInchExchange.js

```diff
@@ -1,38 +1,42 @@
 const Token = artifacts.require("ExpandedERC20");
 const OneSplit = artifacts.require("OneSplit");
 
-const oneSplitAddress = "0xC586BeF4a0992C495Cf22e1aeEE4E446CECDee0E";
-
 // As defined in 1inch
 const ETH_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
 
+const ONE_SPLIT_ADDRESS = "0xC586BeF4a0992C495Cf22e1aeEE4E446CECDee0E";
+
 class OneInchExchange {
   /**
    * @notice Creates OneInchExchange client
-   * @param {Object} args.web3 Web3 instance
-   * @param {Object} args.gasEstimator GasEstimator instance
+   * @param {Object} param - Constructor params
+   * @param {Object} param.web3 - Web3 instance
+   * @param {Object} param.gasEstimator - GasEstimator instance
+   * @param {string} param.oneSplitAddress - Address of the One Split
    * */
-  constructor(args = { web3, gasEstimator }) {
```

Comment:
I know this was here before this PR, but should the `OneInchExchange` not include the logger & associated logging states? Definitely can (and should) be a separate PR but just wanted to ask about this here :)